File: Basic/Loop_and_Count_to_200M.py
import time

start_time = time.time()

# Loop and count to 200 million
count = 0
for i in range(200_000_000):
    count += 1

execution_time = time.time() - start_time
print(f"Python Count: {count}")
print(f"Python Time Taken: {execution_time:.4f} seconds")

File: Loop_and_Count_to_200M.rs
use std::time::Instant;

fn main() {
    let start_time = Instant::now();

    // Loop and count to 200 million
    let mut count = 0;
    for _ in 0..200_000_000 {
        count += 1;
    }

    let execution_time = start_time.elapsed();
    println!("Rust Count: {}", count);
    println!("Rust Time Taken: {:.4?} seconds", execution_time);
}
